src/nodes/chunk_append/planner.c

```diff
@@ -272,10 +273,21 @@ ts_chunk_append_plan_create(PlannerInfo *root, RelOptInfo *rel, CustomPath *path
 				chunk_rt_indexes = lappend_oid(chunk_rt_indexes, scan->scanrelid);
 			}
 		}
+
 		Assert(list_length(cscan->custom_plans) == list_length(chunk_ri_clauses));
 		Assert(list_length(chunk_ri_clauses) == list_length(chunk_rt_indexes));
 	}
 
+	/* only do parent exclusion if not doing chunk-by-chunk exclusion */
```

Comment:
It would be good to have a comment describing what the parent exclusion is, as opposed to chunk-by-chunk exclusion.

Reply:
Put a big comment in chunk_append.c to explain things